app/javascript/src/components/Dashboard/Notes/Note.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+import { MenuVertical, Clock } from "neetoicons";
+import { Typography, Button, Avatar, Dropdown, Tooltip } from "neetoui";
+
+import { calculateCreatedAgo, dateToDayTimeFormat } from "utils/dateTime";
```

Comment:
Check the Slack thread. I have pointed out the issue with code similar to this one.